packages/stateful/voting-module-adapter/adapters/DaoVotingCw4/actions/makeManageMembersAction/ManageMembersComponent.tsx

```diff
@@ -75,8 +77,17 @@ export const ManageMembersComponent: ActionComponent<
       title={t('title.manageMembers')}
     >
       <div className="flex flex-col gap-1">
-        <InputLabel name={t('form.membersToAddOrUpdate')} />
-        <div className="flex flex-col items-stretch gap-1">
+        <div className="flex flex-row gap-2">
+          <InputLabel name={t('form.membersToAddOrUpdate')} />
+          <Tooltip
+            arrow
+            placement="top"
+            title={t('form.membersToAddOrUpdateDescription')}
+          >
+            <Info className="text-text-tertiary" fontSize="small" />
+          </Tooltip>
```

Comment:
can you replace this with the `TooltipInfoIcon` component to keep it consistent :D

we use this pattern all over the place

Reply:
oh hell yeah thanks i was looking for that

Reply:
centralized documentation with screenshots of each component and pointers to where to look will fix this problem 🤦 soon <3